en/howto/reinstall-nix.md

```diff
@@ -0,0 +1,7 @@
+# Reinstall Nix
+
+1. Reboot your system (to ensure that none of the processes are using the `/nix/store` volume)
+1. Uninstall Nix:
+    - Run `/nix/nix-installer uninstall`
+    - If that path above does not exist, [follow these instructions](https://nixos.org/manual/nix/stable/installation/uninstall.html#macos) to manually uninstall Nix.
+3. [[install]]
```

Comment:
@srid Suggesting <https://nixos.asia/en/install> here can lead to folks not using `juspay/nixone`.

I wonder, if we should add a `Note` in <https://nixos.asia/en/install>, asking users to run `juspay/nixone` for one command to setup all things Nix.

Reply:
The hardcoding to `@juspay.in` in `git-email` is one case against adding the note in <https://nixos.asia/en/install>